docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -31,6 +31,83 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 
 **Note:** A full restart is required for Gitea configuration changes to take effect.
 
+## Table of Contents. <!-- omit in toc -->
+
+<!-- Keep this list up to date and the links in it working across rewordings of headings using one of these markdown extensions in your IDE of choice:
```

Comment:
do we have to mention the VSCode plugin everywhere?

Reply:
I was asking myself the same question. Having (potentially a list of) plugins like that everywhere you have a TOC would cause some bloat and repetition. Having it just once (e.g. in a tooling chapter of the contributing docs) would be cleaner. But then I also doubted whether all contributors would actually read and follow those. I for example was not aware of this https://github.com/go-gitea/gitea/tree/master/docs#install

Either way, using Hugo as suggested below renders this question moot.